### PR TITLE
Document new comparison and set membership operators

### PR DESCRIPTION
**Pull request by spjwebster**
...from PR #8

### CODE REVIEW

**Comment by spjwebster** [comment]
@BarnabyShearer would be grateful if you could check this documentation matches your recent PR

File: docs/expressions.rst [hunk 1]
```diff
@@ -47,10 +47,26 @@ that order.
...
+``<``, ``lt``                    Less than                 ``foo < 5``
+``<=``, ``le``, ``≤``            Less than or equal to     ``foo <= 5``
+``is``                           Identity                  ``foo is True``
+``isnot``                        Inverse identity          ``foobar isnot True``
+===============================  ========================  =====================
+
+
+Boolrule also supports extended comparison operators
+
 =======================  ========================  =====================
```
Comment (BarnabyShearer) [inline docs/expressions.rst]: @spjwebster The descriptions look great. However i think you intended to replace the this table.

Comment (spjwebster) [reply docs/expressions.rst]: Ah, yes... I was going to document the new ones separately but realised it didn't make too much sense. Thanks

**Comment by spjwebster** [comment]
great, thanks @BarnabyShearer

**Pull request merged**